qa/workunits/cephtool/test.sh

```diff
@@ -1330,6 +1330,7 @@ function test_mon_pg()
   ceph pg ls
   ceph pg ls 0
   ceph pg ls stale
+  ceph pg ls scrubq
```

Comment:
this would fail the whole test as we have `set -e` at the beginning of this script. maybe you can use `expect_false` here?